packages/framer-motion/src/animation/legacy-popmotion/decay.ts

```diff
@@ -1,29 +1,32 @@
-import { Animation, AnimationState, DecayOptions } from "./types"
+import { DecayOptions } from "../types"
+import { Animation, AnimationState } from "./types"
 
 export function decay({
+    keyframes = [0],
```

Comment:
This looks a bit weird, so I think we should
- Verify an array that contains a zero is actually what you mean here
- Add a comment why this is.

Reply:
I've added a comment explaining this now, to expand on this it's less wrangling to coerce the different formats of `to: V[]` and `to: V, from: V` into a keyframes array as early as possible as this is used by both WAAPI and the majority of our JS animations. It looks a bit funny for `decay` but doing it this way keeps the interface compatible with all the other animations.